test/unit/effects.js

```diff
@@ -2363,4 +2365,266 @@ test( "Show/hide/toggle and display: inline", function() {
+
+	equal( defaultHook.get( fakeTween ), 15, "Gets expected value not defined on style" );
+	equal( cssStub.callCount, 0, "Did not call jQuery.css" );
+
+	fakeTween.prop = "testMissing";
+	equal( defaultHook.get( fakeTween ), 10, "Can get missing property on element" );
+	ok( cssStub.calledWith( testElement, "testMissing", ""), "...using jQuery.css" );
+
+	cssStub.returns( "" );
+	equal( defaultHook.get( fakeTween ), 0, "Uses 0 for empty string" );
+
+	cssStub.returns( "auto" );
+	equal( defaultHook.get( fakeTween ), 0, "Uses 0 for 'auto'" );
+
+	cssStub.returns( null );
+	equal( defaultHook.get( fakeTween ), 0, "Uses 0 for null" );
+
+	cssStub.returns( undefined );
+	equal( defaultHook.get( fakeTween ), 0, "Uses 0 for undefined" );
+
+	cssStub.reset();
+
+	// Setters
+	styleStub = this.sandbox.stub( jQuery, "style" );
+	fakeTween.prop = "height";
+
+	defaultHook.set( fakeTween );
+	ok( styleStub.calledWith( testElement, "height", "10px" ), "Calls jQuery.style with elem, prop, now+unit" );
+
+	styleStub.reset();
+	fakeTween.prop = "testMissing";
+
+	defaultHook.set( fakeTween );
+	equal( styleStub.callCount, 0, "Did not call jQuery.style for non css property" );
+	equal( testElement.testMissing, 10, "Instead, set value on element directly" );
+
+	jQuery.cssHooks.testMissing = jQuery.noop;
+	fakeTween.now = 11;
+
+	defaultHook.set( fakeTween );
+	delete jQuery.cssHooks.testMissing;
+
+	ok( styleStub.calledWith( testElement, "testMissing", "11px" ), "Presence of cssHooks causes jQuery.style with elem, prop, now+unit" );
+	equal( testElement.testMissing, 10, "And value was unchanged" );
+
+	stepSpy = jQuery.fx.step.test = this.sandbox.spy();
+	styleStub.reset();
+
+	fakeTween.prop = "test";
+	defaultHook.set( fakeTween );
+	ok( stepSpy.calledWith( fakeTween ), "Step function called with Tween" );
+	equal( styleStub.callCount, 0, "Did not call jQuery.style" );
+});
+
+test( "jQuery.Tween - Plain Object", function() {
+	expect( 13 );
+	var testObject = { test: 100 },
+		testOptions = { duration: 100 },
+		tween, easingSpy;
+
+	tween = jQuery.Tween( testObject, testOptions, "test", 0, "linear" );
+	equal( tween.elem, testObject, "Sets .element" );
+	equal( tween.options, testOptions, "sets .options" );
+	equal( tween.prop, "test", "sets .prop" );
+	equal( tween.end, 0, "sets .end" );
+
+	equal( tween.easing, "linear", "sets .easing when provided" );
+
+	equal( tween.start, 100, "Reads .start value during construction" );
+	equal( tween.now, 100, "Reads .now value during construction" );
+
+	easingSpy = this.sandbox.spy( jQuery.easing, "linear" );
+
+	equal( tween.run(0.1), tween, ".run() returns this" );
+
+	equal( tween.now, 90, "Calculated tween" );
+
+	ok( easingSpy.calledWith( 0.1, 0.1 * testOptions.duration, 0, 1, testOptions.duration ), "...using jQuery.easing.linear with back-compat arguments" );
+	equal( testObject.test, 90, "Set value" );
+
+	tween.run(1);
+	equal( testObject.test, 0, "Checking another value");
+
+	tween.run(0);
```

Comment:
Because we use an old JSCS version and we do that because a newer one would blow up on 90% of our test code, it doesn't follow our own style guide for historical reasons.

AFAIK it mostly waits until JSCS learns how to auto-correct incorrectly formatted code. We'll then pass it on our test code and all will start conforming.